core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -1285,7 +1285,6 @@ ExWorkProcRetcode ExHbaseAccessBulkLoadPrepSQTcb::work()
       matches_ = 0;
       currRowNum_ = 0;
       numRetries_ = 0;
-      hFileParamsInitialized_ = FALSE;
```

Comment:
samplingRate will be >0 only if the syntax to do it is used. Yes, the flush refers to our own buffers. The HFile flushing is separate. I removed the comment.